tests/css_atrule/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,98 +1,2394 @@
+@each $animal in puma, sea-slug, egret, salamander {}
+@each $animal in puma,sea-slug,egret,salamander{}
+@each  $animal  in  puma  ,  sea-slug  ,  egret  ,  salamander  {}
+@each $animal in
+    puma, sea-slug, egret, salamander {}
+@each
+    $animal
+    in
+    puma
+    ,
+    sea-slug
+    ,
+    egret
+    ,
+    salamander
+    {
+    }
+@each
+$animal
+in
+puma
+,
+sea-slug
+,
+egret
+,
+salamander
+{
+}
+@each
+
+$animal
+
+in
+
+puma
+
+,
+
+sea-slug
+
+,
+
+egret
+
+,
+
+salamander
+
+{
+
+}
+@each $animal in ((puma), (sea-slug), (egret), (salamander)) {}
+@each $animal in((puma),(sea-slug),(egret),(salamander)){}
+@each  $animal  in  (  (  puma  )  ,  (  sea-slug  )  ,  (  egret  )  ,  (  salamander  )  )  {}
+@each $animal
+    in ((puma), (sea-slug), (egret), (salamander)) {}
+@each
+    $animal
+    in (
+        (puma),
+        (sea-slug),
+        (egret),
+        (salamander)
+    ) {  }
+@each
+    $animal
+    in
+    (
+    (
+    puma
+    )
+    ,
+    (
+    sea-slug
+    )
+    ,
+    (
+    egret
+    )
+    ,
+    (
+    salamander
+    )
+    )
+    {
+    }
+@each
+
+    $animal
+
+    in
+
+    (
+
+    (
+
+    puma
+
+    )
+
+    ,
+
+    (
+
+    sea-slug
+
+    )
+
+    ,
+
+    (
+
+    egret
+
+    )
+
+    ,
+
+    (
+
+    salamander
+
+    )
+
+    )
+
+    {
+
+    }
+@each $animal, $color, $cursor in (puma, black, default), (sea-slug, blue, pointer), (egret, white, move) {}
+@each $animal,$color,$cursor in (puma,black,default),(sea-slug,blue,pointer),(egret,white,move){}
+@each $animal, $color, $cursor in
+    (puma, black, default),
+    (sea-slug, blue, pointer),
+    (egret, white, move) {}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+@each $animal in puma, sea-slug, egret, salamander {
+}
+@each $animal in puma, sea-slug, egret, salamander {
+}
+@each $animal in puma, sea-slug, egret, salamander {
+}
+@each $animal in puma, sea-slug, egret, salamander {
+}
+@each $animal in puma, sea-slug, egret, salamander {
+}
+@each $animal in puma, sea-slug, egret, salamander {
+}
+@each $animal in puma, sea-slug, egret, salamander {
+}
+@each $animal in ((puma),  (sea-slug),  (egret),  (salamander)) {
+}
+@each $animal in((puma), (sea-slug), (egret), (salamander)) {
+}
+@each $animal in ( (puma),  (sea-slug),  (egret),  (salamander)) {
+}
+@each $animal in ((puma),  (sea-slug),  (egret),  (salamander)) {
+}
+@each $animal in ( (puma),  (sea-slug),  (egret),  (salamander)) {
+}
+@each $animal in ( (puma),  (sea-slug),  (egret),  (salamander)) {
+}
+@each $animal in ( (puma),  (sea-slug),  (egret),  (salamander)) {
+}
+@each $animal, $color, $cursor in (puma, black, default),
+   (sea-slug, blue, pointer),  (egret, white, move) {
+}
+@each $animal, $color, $cursor in (puma, black, default),
+  (sea-slug, blue, pointer), (egret, white, move) {
+}
+@each $animal, $color, $cursor in (puma, black, default),
+   (sea-slug, blue, pointer),  (egret, white, move) {
+}
```

Comment:
```scss
@each $animal, $color, $cursor in (
  (puma, black, default),
  (sea-slug, blue, pointer),
  (egret, white, move)
) {
}
```

Reply:
@j-f1 do you think it normal to add `(` and `)` if they are not necessary (we don't output `(` and `)` they are unnecessary in js)?